docs/FAQ.md

```diff
@@ -33,27 +33,23 @@ If you are using Apache APISIX below v2.0, be aware that the data from the etcd
 
 ### 4. After modifying the plugin schema or creating a custom plugin in Apache APISIX, why can't I find it on the dashboard?
 
-Since the Dashboard caches the jsonschema data of the plugins in Apache APISIX, you need to synchronize the data in the Dashboard after you create your custom plugins in Apache APISIX, which currently **only supports manual operation**, this issue will be optimized in the following versions. Please follow the following guide.
+Since the Dashboard caches the jsonschema data of the plugins in Apache APISIX, you need to synchronize the data in the Dashboard after you create your custom plugins in Apache APISIX, which currently **only supports manual operation**. Please follow the following guide.
 
-1. Install [Lua](https://www.lua.org/download.html) and `zip`.
+1. Confirm that your APISIX is running and has enabled control API (enabled by default and only runs local access)
+Refer to the beginning in:
+https://github.com/apache/apisix/blob/master/doc/control-api.md
 
-2. Execute the following commands.
+2. Execute the following commands to export jsonchema on your APISIX server (if it is configured for non-local access, it does not need to be executed on your APISIX server, and the access IP and port should be modified accordingly)
 
 ```sh
-# `$version` is the version number of Apache APISIX, e.g. master or 2.1.
-$ api/build-tools/schema-sync.sh $version
+curl 127.0.0.1:9090/v1/schema> schema.json
 ```
 
-If you have a custom plugin, make sure it is in the `apisix` directory and use the following command.
+Refer to https://github.com/apache/apisix/blob/master/doc/control-api.md#get-v1schema
 
-```sh
-$ api/build-tools/schema-sync.sh /path/to/apisix
-
-# e.g
-$ api/build-tools/schema-sync.sh /usr/local/apisix
-```
+3. Copy the exported schema.json to the conf directory in the Dashboard working directory
```

Comment:
how about add a link to `conf directory` to tell user where it is?

Reply:
sure, thanks!